keras_nlp/samplers/sampler.py

```diff
@@ -39,6 +40,17 @@
     from_logits: bool, defaults to True. Indicate if the `token_probability_fn`
         returns logits. If False, `token_probability_fn` returns probability
         distributions.
+    cache: a dense int tensor, the cache used in decoding. The cache
+        stores the K and V of each
```

Comment:
K, V -> key, value

Also , rather than point to `CachedMultiHeadAttention`, I would give a high level description of the function of the cache here.

```
A cache of intermediate key and value tensor computed by each
decoder self-attention layer. These values will only be computed
once for each new token in the generated sequence.
```